docs/yaml.md

```diff
@@ -27,6 +27,92 @@ This stanza describes how to build the Docker image your model runs in. It conta
 
 Cog automatically picks the correct version of CUDA to install, but this lets you override it for whatever reason.
 
+### `environment`
```

Comment:
"View File" for easier reviewing:

https://github.com/hangtwenty/cog/blob/cog-environment-variables/docs/yaml.md#environment